articles/quickstart/webapp/nodejs/01-login.md

```diff
@@ -18,17 +18,41 @@ github:
 
 ## Configure Node.js to use Auth0
 
+### Create the .env file
+
+Create the `.env` file in the root of your app and add your Auth0 variables and values to it.
+
+```
+# .env
+AUTH0_CLIENT_ID=${account.clientId}
+AUTH0_DOMAIN=${account.namespace}
```

Comment:
I'm not sure if you're dealing with Custom Domains in the QS right now, or how, but just as a heads up, if you are, you might want to make a note here (` // Or, your custom domain (link) `) since `account.namespace` will just fill in their auth0 domain.

Reply:
I just talked with @aaguiarz. This is an issue that might need to be addressed in general.